### ISSUE TITLE
Another Error: Tesla sign in page error - Unknown

### ISSUE THREAD

**Issue by IMgoRt** [fredli74/fetch-tesla-token#3]
Looks like Tesla has enabled captcha again:

```
TeslaAuthUnauthorized [Error]: Tesla sign in page error - Unknown
    at Object.authenticate (C:\Users\david\fetch-tesla-token\teslaAuth.js:237:13)
    at processTicksAndRejections (node:internal/process/task_queues:94:5)
    at async Object.exports.handler (C:\Users\david\fetch-tesla-token\functions\proxy.js:18:25)
    at async IncomingMessage.<anonymous> (C:\Users\david\fetch-tesla-token\server.js:26:27)
```

Thanks

**Comment by fredli74** [comment]
Thank you. Yeah, this kinda throws a wrench into the simple API I have today, as it will require a challenge-response kind of interaction. 

I will find a fix for this as my users on smartcharge.dev needs it. But I might come up with a different solution. How are you using this API/project today?

**Comment by dranreb38** [comment]
+1 idem 
:(

**Comment by fredli74** [comment]
Just pushed a new updated version. Let me know if it works.

Unfortunately, this made the authentication flow a lot more complicated if you are using the API. It now goes something like this:
```js
const { session, htmlForm } = await teslaAuth.newSession();

htmlForm.identity = input.email;
htmlForm.credential = input.password;

if (htmlForm.captcha !== undefined) {
  if (input.captcha) {
    htmlForm.captcha = input.captcha;
  } else {
    const captchaURI = await teslaAuth.getCaptcha(session);
    // User intervention needed to solve the captcha
    return captchaURI;
  }
}

if (input.mfa) {
  await teslaAuth.validateFactors(session, input.mfa, htmlForm.transaction_id);
}

const authCode = await teslaAuth.authenticate(session, htmlForm);

const bearerToken = await teslaAuth.bearerToken(authCode, session.codeVerifier);

const ownerToken = await teslaAuth.ownerapiToken(bearerToken.access_token);
```

**Comment by fredli74** [comment]
It might be possible that you can always request a captcha to streamline the flow in two steps. But I have not tested it, as I went the totally different way for my service and only asking for captcha and mfa if it is required.

```js
const { session, htmlForm } = await teslaAuth.newSession();
const captchaURI = await teslaAuth.getCaptcha(session);

// Show captcha and ask user for input

htmlForm.identity = input.email;
htmlForm.credential = input.password;
htmlForm.captcha = input.captcha;
...
```

**Comment by IMgoRt** [comment]
Working for me, including captcha which is randomly back on today for me

**Comment by dranreb38** [comment]
It's ok for me , but error error 400 Bad request (with MFA) .
![Screenshot_252](https://user-images.githubusercontent.com/16446911/127019107-cc8fa77f-e876-4d3f-9d09-cfef950776f9.png)

**Comment by IMgoRt** [comment]
> Working for me, including captcha which is randomly back on today for me

Mine also has MFA enabled, passed without issue

**Comment by fredli74** [comment]
Solved in f0daaeacde7542d03bf8f4f942431bb880dab7ea
